modules/persistent-storage-csi-secrets-store-driver-overview.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-secrets-store-driver-overview_{context}"]
+= Overview
+
+{product-title} secrets are stored with just base64 encoding, which can be easily decoded. Etcd provides encryption at rest for these secrets, but when secrets are retrieved they are decrypted and presented to the user. If RBAC is not set up on your cluster properly, anyone with API access can retrieve or modify a secret, and so can anyone with access to etcd. Additionally, anyone who is authorized to create a pod in a namespace can use that access to read any secret in that namespace. 
+
+For secure storage and management of your secrets, the {product-title} Secrets Store Container Storage Interface (CSI) Driver Operator allows you to mount secrets from external secret management system, such as AWS Secrets Manager or Azure Key Vault using a provider plugin. Applications can use the secret, but they do not persist on the system after the application pod is destroyed.
+
+The Secrets Store CSI Driver Operator, `secrets-store.csi.k8s.io`, allows {product-title} to mount multiple secrets, keys, and certs stored in enterprise-grade external secrets stores into pods as a volume. After the volume is attached, the data in it is mounted into the container's file system.
+
+:FeatureName: Secrets Store CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The Secrets Store CSI Driver is a daemonset that facilitates communication with every instance of Kubelet. Each driver pod has the following containers:
+
+* `node-driver-registrar`: Responsible for registering the CSI driver with Kubelet so that it knows which Unix domain socket to issue the CSI calls on.
+
+* `secrets-store`: Implements the CSI Node service gRPC services described in the CSI specification. It is responsible for mounting and unmounting the volumes during pod creation and deletion.
+
+* `liveness-probe`: Responsible for monitoring the health of the CSI driver and reports to Kubernetes. This enables Kubernetes to automatically detect issues with the driver, and then restart the pod to try and fix the issue.
+
+The Secrets Store CSI Driver Operator communicates with the provider using gRPC to fetch the mount contents from external Secrets Store.
+
+Supported providers include:
```

Comment:
Would you mind removing this section? I'm planning on having a small section on this, and also which ones we support this release will depend on what QE tests, still TBD.